os/os.proto

```diff
@@ -7,7 +7,7 @@ import "github.com/openconfig/gnoi/types/types.proto";
 
 option go_package = "github.com/openconfig/gnoi/os";
 
-option (types.gnoi_version) = "0.1.0";
+option (types.gnoi_version) = "0.2.0";
```

Comment:
assuming the suggestion below, I am thinking this would be a micro bump and opposed to minor.

Reply:
Done